test/test_regression.py

```diff
@@ -47,21 +57,24 @@ def get_script_module(script_name):
     return imp.load_module(script_name, file, pathname, desc)
 
 
-def get_csv_md5(dataset, engines, tmpdir):
+def get_csv_md5(dataset, engines, tmpdir, install_function, cofig):
```

Comment:
It looks like we're only passing in one engine at a time in which case change `engines` to `engine`.